HW04/aleksei_zhuchkov/lib/student.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class Student
+  attr_accessor :storage_homework, :storage_notification_for_student, :storage_notification_for_mentor
+  attr_reader :name, :surname
+
+  def initialize(student)
+    @name = student[:name].capitalize
+    @surname = student[:surname].capitalize
+    @storage_homework = []
+    @storage_notification_for_student = []
+    @storage_notification_for_mentor = []
+  end
+
+  def add_homework_to_student_storage(homework)
+    @storage_homework << homework
+  end
+
+  def notifications
```

Comment:
I don't think any notification logic is the responsibility of the Student class

Reply:
I hope, I was understand you right. I changed my code, pls check.